math/main.go

```diff
@@ -0,0 +1,10 @@
+package math
+
+// Min returns the smallest number of the two inputs
```

Comment:
It's not worthwhile putting this in its own package by itself, particularly since it has the same name as a stdlib package. I wouldn't bother, especially as you call this precisely once fwict.